ai/tests/tools/test_gift_tax_calculator.py

```
@@ -0,0 +1,146 @@
+            gift_property_value=500_000_000,
+            secured_debt=200_000_000,
+        )
+
+        # When
+        result = calculate_gift_tax_simple(**input_data.model_dump())
+
+        # Then
+        assert result["gift_value"] == 300_000_000  # 5억 - 2억
+        assert result["total_deduction"] == 50_000_000
+        assert result["taxable_base"] == 250_000_000
+        assert result["calculated_tax"] == 40_000_000  # 2.5억 × 20% - 1천만 = 4천만
```

Ruff (0.14.0)

Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)